examples/webxr_vr_teleport.html

```diff
@@ -0,0 +1,278 @@
+				controllerGrip2.add( controllerModelFactory.createControllerModel( controllerGrip2 ) );
+				scene.add( controllerGrip2 );
+
+				//
+
+				window.addEventListener( 'resize', onWindowResize, false );
+
+			}
+
+			function buildController( data ) {
+
+				let geometry, material;
+
+				switch ( data.targetRayMode ) {
+
+					case 'tracked-pointer':
+
+						geometry = new THREE.BufferGeometry();
+						geometry.setAttribute( 'position', new THREE.Float32BufferAttribute( [ 0, 0, 0, 0, 0, - 1 ], 3 ) );
+						geometry.setAttribute( 'color', new THREE.Float32BufferAttribute( [ 0.5, 0.5, 0.5, 0, 0, 0 ], 3 ) );
+
+						material = new THREE.LineBasicMaterial( { vertexColors: true, blending: THREE.AdditiveBlending } );
+
+						return new THREE.Line( geometry, material );
+
+					case 'gaze':
+
+						geometry = new THREE.RingGeometry( 0.02, 0.04, 32 ).translate( 0, 0, - 1 );
+						material = new THREE.MeshBasicMaterial( { opacity: 0.5, transparent: true } );
+						return new THREE.Mesh( geometry, material );
+
+				}
+
+			}
+
+			function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+			}
+
+			function handleController( controller ) {
+
+				if ( controller.userData.isSelecting ) {
+
+					const object = room.children[ count ++ ];
+
+					object.position.copy( controller.position );
+					object.userData.velocity.x = ( Math.random() - 0.5 ) * 3;
+					object.userData.velocity.y = ( Math.random() - 0.5 ) * 3;
+					object.userData.velocity.z = ( Math.random() - 9 );
+					object.userData.velocity.applyQuaternion( controller.quaternion );
+
+					if ( count === room.children.length ) count = 0;
+
+				}
+
+			}
+
+			//
+
+			function animate() {
+
+				renderer.setAnimationLoop( render );
+
+			}
+
+			function render() {
+
+				INTERSECTED = undefined;
+
+				if ( controller1.userData.isSelecting === true ) {
+
+					tempMatrix.identity().extractRotation( controller1.matrixWorld );
+
+					raycaster.ray.origin.setFromMatrixPosition( controller1.matrixWorld );
+					raycaster.ray.direction.set( 0, 0, - 1 ).applyMatrix4( tempMatrix );
+
+					const intersects = raycaster.intersectObjects( [ floor ] );
+
+					if ( intersects.length > 0 ) {
+
+						INTERSECTED = intersects[ 0 ].point;
+
+					}
+
+				} else if ( controller2.userData.isSelecting === true ) {
+
+					tempMatrix.identity().extractRotation( controller2.matrixWorld );
+
+					raycaster.ray.origin.setFromMatrixPosition( controller2.matrixWorld );
+					raycaster.ray.direction.set( 0, 0, - 1 ).applyMatrix4( tempMatrix );
+
+					const intersects = raycaster.intersectObjects( [ floor ] );
+
+					if ( intersects.length > 0 ) {
+
+						INTERSECTED = intersects[ 0 ].point;
+
+					}
+
+				}
+
+				if ( INTERSECTED ) marker.position.copy( INTERSECTED );
+
+				marker.visible = !!INTERSECTED;
```

Comment:
Use `INTERSECTED !== undefined` instead of `!!INTERSECTED` (also it is better to rename it to `INTERSECTION`, or just `intersection`).

Reply:
Personally I fins `!!INTERSECTED` much cleaner but no objections here. Changed.

Reply:
`if ( INTERSECTION )` can also be changed to `if ( INTERSECTION !== undefined )`.